components/telemetry-operator/internal/fluentbit/variable_validator.go

```diff
@@ -0,0 +1,63 @@
+package fluentbit
```

Comment:
This is not really related to fluentbit. How about moving all the validators to an own `validation` package?